tekton/utils/pipeline_run_builder.go

```diff
@@ -193,6 +193,30 @@ func (b *PipelineRunBuilder) WithParamsFromConfigMap(configMap *corev1.ConfigMap
 func (b *PipelineRunBuilder) WithPipelineRef(pipelineRef *tektonv1.PipelineRef) *PipelineRunBuilder {
 	b.pipelineRun.Spec.PipelineRef = pipelineRef
 
+	if pipelineRef.Resolver == "git" {
```

Comment:
This part is still untested I think

Reply:
I copied the test we had before (`It("adds the taskGit pipeline parameters to the PipelineRun object when using a git resolver", func() {`. Not sure what you mean.

Reply:
`adds the taskGit` does not exist in `tekton/utils/pipeline_run_builder_test.go` 
?

Reply:
Maybe it wasn't pushed. It's there now.